lib/ts/recipe/session/recipeImplementation.ts

```diff
@@ -167,26 +167,26 @@ export default function getRecipeInterface(
             let doAntiCsrfCheck = options !== undefined ? options.antiCsrfCheck : undefined;
             const transferMethod = config.getTokenTransferMethod({ req, userContext });
 
-            let idRefreshToken = getToken(config, req, "idRefresh", userContext, transferMethod);
```

Comment:
do we not need to check if the idrefreshtoken is there in the input during session refreshing to know if the old method was used or not? - is there nothing special that we need to do for migrating from older sessions to newer ones?